fabric-resource-loader-v0/src/testmod/java/net/fabricmc/fabric/test/resource/loader/LanguageTestMod.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.resource.loader;
+
+import net.minecraft.text.Text;
+
+import net.fabricmc.api.ModInitializer;
+
+public class LanguageTestMod implements ModInitializer {
+	@Override
+	public void onInitialize() {
+		testTranslationLoaded();
+	}
+
+	private static void testTranslationLoaded() {
```

Comment:
This is crashing on the client, I think im going to make the testmod and mixin only apply on the server.

We really really need some client test coverage on CI.

Reply:
Oh, apologies! I thought I'd tested on the client, but clearly not got that bit right.

Thank you for fixing it :).